chowpronto_frontend/src/patron/pages/menu/components/MenuItem.js

```diff
@@ -22,7 +23,7 @@ const MenuItem = (props) => {
         <HeaderRight></HeaderRight>
       </FooterBar>
       <ClearButton
-        onClick={() => updateBasket({ type: "add_item", item: props })}
+        onClick={() => dispatch({ type: "add_item_to_basket", item: props })}
```

Comment:
I was mislead by ClearButton name :). This button add item to the basket and I thought it was clearing(removing) something. :)